src/commands/android/utils/common.ts

```diff
@@ -152,3 +155,57 @@ export const downloadFirefoxAndroid = async (version: string) => {
 
   return await downloadWithProgressBar(apkDownloadUrl, tempdir);
 };
+
+export const getSdkRootFromEnv = (rootDir: string, androidHomeInGlobalEnv: boolean): string => {
+  Logger.log('Checking the value of ANDROID_HOME environment variable...');
+
+  const androidHome = process.env.ANDROID_HOME;
+  const fromDotEnv = androidHomeInGlobalEnv ? '' : ' (taken from .env)';
+
+  if (androidHome) {
+    const androidHomeFinal = untildify(androidHome);
+
+    const androidHomeAbsolute = path.resolve(rootDir, androidHomeFinal);
```

Comment:
Another very minor thing, we should probably change the `rootDir` name here with `cwd`, because 'rootDir' can be confused with the SDK root dir but what we mean by rootDir here is actually the current working directory, or the root directory of the project.
```suggestion
export const getSdkRootFromEnv = (cwd: string, androidHomeInGlobalEnv: boolean): string => {
  Logger.log('Checking the value of ANDROID_HOME environment variable...');

  const androidHome = process.env.ANDROID_HOME;
  const fromDotEnv = androidHomeInGlobalEnv ? '' : ' (taken from .env)';

  if (androidHome) {
    const androidHomeFinal = untildify(androidHome);

    const androidHomeAbsolute = path.resolve(cwd, androidHomeFinal);
```